waves/wallet/src/wallet/make_create_swap_payload.rs

```diff
@@ -52,13 +88,26 @@ pub async fn make_create_sell_swap_payload(
     })
     .await?;
 
-    // Bob currently hardcodes a fee-rate of 1 sat / vbyte, hence there is no need for us to perform fee estimation.
-    // Later on, both parties should probably agree on a block-target and use the same estimation service.
-    let bobs_fee_rate = Amount::from_sat(1);
-    let fee_offset = calculate_fee_offset(bobs_fee_rate);
+    let (bobs_fee_rate, fee_offset) = if fee_asset == sell_asset {
+        // Bob currently hardcodes a fee-rate of 1 sat / vbyte, hence
+        // there is no need for us to perform fee estimation. Later
+        // on, both parties should probably agree on a block-target
+        // and use the same estimation service.
+        let bobs_fee_rate = Amount::from_sat(1);
+        let fee_offset = calculate_fee_offset(bobs_fee_rate);
+
+        (bobs_fee_rate, fee_offset)
```

Comment:
I just realized that we might have a very small issue: 
If Alice sells BTC we check on the UI if her `sell_amount < balance`. 
If I see it correctly, then we add the `fee_offset` on top of `sell_amount` which means the coin selection algorithm might fail if she tries to sell her full balance.

Reply:
Yeah, you're right. I'll create an issue!

Reply:
Recorded here: https://github.com/comit-network/waves/issues/137.